protocol/testing/genesis.sh

```diff
@@ -969,7 +975,8 @@ function edit_genesis() {
 	dydx_exchange_config_json=$(cat "$EXCHANGE_CONFIG_JSON_DIR/dydx_exchange_config.json" | jq -c '.')
 	dasel put -t string -f "$GENESIS" '.app_state.prices.market_params.[34].exchange_config_json' -v "$dydx_exchange_config_json"
 
-	bridge_module_account_balance=$TOTAL_NATIVE_TOKEN_SUPPLY
+	# Initialize bridge module account balance as total native token supply minus rewards vester account balance.
+	bridge_module_account_balance=$(echo "$TOTAL_NATIVE_TOKEN_SUPPLY - $REWARDS_VESTER_ACCOUNT_BALANCE" | bc)
```

Comment:
Instead of initializing `module account balance = total_supply - vester_balance`, can we do decrement `bridge_module_account_balance balance` around line 1014, similar to how to decrement other type of balances?